src/convolution.jl

```diff
@@ -47,6 +50,20 @@ end
 convolve(d1::Poisson, d2::Poisson) =  Poisson(d1.λ + d2.λ)
 
 
+
+function convolve(d1::DiscreteNonParametric, d2::DiscreteNonParametric)
+    support_conv = vcat([d1.support .+ v for v in d2.support]...) |> unique 
```

Comment:
This seems a bit inefficient: due to the allocations, splatting and constructing the full vector. Probably better to use a `Set`:

```suggestion
    support_conv = collect(Set(v1 + v2 for v1 in support(d1), v2 in support(d2)))
```